app/main/lib/similarity_helpers.py

```diff
@@ -26,11 +26,11 @@ def drop_context_from_record(record, context):
     deleted = True
     return deleted
 
-def get_context_query(context, value_as_json=False, vars_as_hash=True):
+def get_context_query(context, value_as_json=True, vars_as_hash=True):
     context_query = []
     context_hash = {}
     for key, value in context.items():
-        if key not in ["project_media_id", "temporary_media"]:
+        if key not in ["project_media_id", "temporary_media", "content_type"]:
```

Comment:
It's not the specific changes made, but it seems like Line 38 is a bug here. 

```
                for i,v in enumerate(value):
                    if value_as_json:
                        context_clause += "context @> '[{\""+key+"\": "+json.dumps(value)+"}]'"
```

Shouldn't `value` be `v` in that line?

Reply:
You're right on this one - I don't think this particular case comes up anywhere but good catch